Makefile

```diff
@@ -27,7 +27,7 @@ C_BINDING_CLEAN = $(foreach dir,$(C_BINDING_DIRS),$(dir)Makefile $(dir).qmake.st
 # apply formatting / style guidelines, and build the rust project
 main:
 	cargo fmt
-	cargo clippy -- --deny clippy
+	cargo clippy -- --warn clippy
```

Comment:
Why would you do that? We want the process to fail if it doesn't comply

Reply:
I want to be able to put returns when it makes the code more understandable and clippy is preventing me from doing that, so I'm trying to make clippy less annoying. To my knowledge, there is no agreement to what the code is suppose to comply to.